config/envs/default.py

```diff
@@ -0,0 +1,11 @@
+from os import environ
+from pathlib import Path
+
+from fsd_utils import configclass
+
+
+@configclass
+class DefaultConfig(object):
+    FLASK_ROOT = str(Path(__file__).parent.parent.parent)
+
+    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL")
```

Comment:
As it stands now this will break the AWS environment as it needs the temp file path which is removed by this PR, please restore that functionality, i.e. make this something like:
```
    SQLALCHEMY_DATABASE_URI = environ.get("DATABASE_URL", f"sqlite:///{tempfile.gettempdir()}/sqlite.db")
```

Reply:
👍 nice spot